.github/workflows/metrics.yml

```diff
@@ -34,13 +34,15 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
     - name: Collect metrics
       run: |
+        echo '${{ secrets.GCP_CONNECTION }}' > $(pwd)/creds.json
         docker run \
         -e BQ_PROJECT_ID=${{ secrets.BQ_PROJECT_ID }} \
         -e GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} \
         -e COBRA_COMMAND=export \
-        -e 'CONNECTION_KEY=${{ secrets.GCP_CONNECTION }}' \
+        -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/creds.json \
```

Comment:
i'm assuming this path & file exist from one of the packages you install in your Dockerfile change?

Reply:
if you look couple lines below that we create the docker volume, which copies over the file echoed to prior to the docker run command
this is the suggested use as per the readme and also saves the pain of passing in an unformatted service key :)

Reply:
ah missed that. Perfect